frontend/src/lib/forms/OrgRoleSelect.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+  import { OrgRole } from '$lib/gql/types';
+  import t from '$lib/i18n';
+
+  export let id = 'role';
+  export let value: OrgRole;
+  export let verbose = false;
+</script>
+
+<select {id} bind:value class="select select-bordered" on:change>
```

Comment:
This no longer has a label in the add member modal.

Reply:
Yep, which is why I had the `{#if withLabel}` param earlier which disappeared in one of the refactors. Since we're going to be moving it into the edit-user modal and NOT sticking the select into the table, I'll go ahead and put the label back; the whole reason for its removal was so that the select-in-the-table thing would look good (c.f. gifs above).

I still like the select-in-the-table idea, and I might try to bring it back later on with an edit icon (pencil) that appears on hover and turns the text box into a role select. BUT... now's not the time to get fancy with this, so I'll put that off until later.

Reply:
Fixed in commit a0b3bf1b where I move to using the modal, rather than select-in-table, to more closely parallel the project page.